assets/monitoring-plugin/deployment.yaml

```diff
@@ -44,7 +44,27 @@ spec:
             topologyKey: kubernetes.io/hostname
       automountServiceAccountToken: false
       containers:
-      - image: quay.io/openshift/origin-monitoring-plugin:1.0.0
+      - command:
+        - /bin/sh
+        - -c
+        - |
+          # TODO: Maybe just use IPV4 if empty?
+          if [ -z "$POD_IP" ]; then
+            echo "POD_IP env variable is not set or is empty"
+            exit 1
+          elif echo "$POD_IP" | grep -qE '^([0-9]{1,3}\.){3}[0-9]{1,3}$'; then
+            LISTEN_ADDR_PORT="9943"
+          else
+            LISTEN_ADDR_PORT="[::]:9443"
+          fi
+          sed "s/LISTEN_ADDR_PORT/$LISTEN_ADDR_PORT/g" /etc/nginx/nginx.conf > /tmp/nginx.conf
+          exec nginx -c /tmp/nginx.conf -g 'daemon off;'
+        env:
+        - name: POD_IP
+          valueFrom:
+            fieldRef:
+              fieldfPath: status.podIP
```

Comment:
`fieldPath` (extra `f` this is for not adding a remote haha)

Reply:
just noticed this as debugging the PR lol 🗡️